torch/distributed/_tensor/placement_types.py

```diff
@@ -39,7 +39,7 @@ def _split_tensor(
         *,
         with_padding: bool = True,
         contiguous: bool = True,
-    ) -> Tuple[List[torch.Tensor], int]:
+    ) -> Tuple[List[torch.Tensor], int, List[int]]:
```

Comment:
so it looks like all the callsites (except `test_device_mesh.py` don't need the second return argument anymore (i think it's because this embeds in the last return argument). Shall we delete the second return arg and return two args `Tuple[List[torch.Tensor], List[int]]` instead?

Reply:
Good point! Actually thinking about the same thing!